taskcluster/script.py

```diff
@@ -151,6 +155,14 @@ def main():
     except subprocess.CalledProcessError as e:
         print('{} attempting netstat'.format(e))
 
+    # enable charging on device
+    #   see https://bugzilla.mozilla.org/show_bug.cgi?id=1565324
+    # TODO: detect the device type
+    p2_cmd = "adb shell su -c 'echo 0 > /sys/class/power_supply/battery/input_suspend'"
+    run_cmd(p2_cmd)
+    g5_cmd = "adb shell su -c 'echo 1 > /sys/class/power_supply/battery/charging_enabled'"
+    run_cmd(g5_cmd)
+
```

Comment:
We should use ADBDevice here. We have it available. 

You can get [device names](https://searchfox.org/mozilla-central/source/testing/raptor/raptor/performance_tuning.py#117) and use shell_bool to set the file contents like

```
self.device.shell_bool("echo %s > %s" % (value, path), root=True):
```

Reply:
OK, changed.